package.json

```diff
@@ -98,7 +98,7 @@
     "css-loader": "^2.0.0",
     "css-mediaquery": "^0.1.2",
     "danger": "^7.0.12",
-    "date-fns": "^2.0.0-alpha.21",
+    "date-fns": "^2.0.0-alpha.33",
```

Comment:
Could you revert this change? It seems to create a duplication in the yarn.lock.

Reply:
I can take care of that if there are some important bug fixes included.

Reply:
> Could you revert this change? It seems to create a duplication in the yarn.lock.

Do you want me to revert it back. Maybe we can fix it and figure out how to remove the old version of date-fns in yarn.lock

Reply:
You can use https://github.com/atlassian/yarn-deduplicate to deduplicate the yarn.lock.

Reply:
Okay I'll use it and try to remove the duplicates. :-)